README.md

```diff
@@ -30,6 +30,27 @@ most of the contents are still valid and very useful:
 
 Developers and contributors: please read our [Contribution Guidelines](./CONTRIBUTING.md).
 
+## Building the Dsbulk 
```

Comment:
The description below is more about installing the tool rather than building it. So it could be `Installing Dsbulk` or similar.